WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -691,16 +691,10 @@ - (UITableViewCell *)configureMetaPostMetaCellForIndexPath:(NSIndexPath *)indexP
     } else if (row == PostSettingsRowPublishDate) {
         // Publish date
         cell = [self getWPTableViewDisclosureCell];
-        if (self.apost.dateCreated && ![self.apost shouldPublishImmediately]) {
-            if ([self.apost hasFuturePublishDate]) {
-                cell.textLabel.text = NSLocalizedString(@"Scheduled for", @"Scheduled for [date]");
-            } else {
-                cell.textLabel.text = NSLocalizedString(@"Published on", @"Published on [date]");
-            }
-
+        cell.textLabel.text = NSLocalizedString(@"Publish Date", @"Label for the publish date button.");
```

Comment:
Momo, I think you also suggested this. This is so much clearer than trying to guess what to show based on state.